binCompatTest/src/main/scala/catsBC/MimaExceptions.scala

```diff
@@ -19,7 +19,7 @@ object MimaExceptions {
     cats.data.Kleisli.catsDataCommutativeFlatMapForKleisli[Option, Int],
     cats.data.IRWST.catsDataStrongForIRWST[List, Int, Int, Int],
     cats.data.OptionT.catsDataMonadErrorMonadForOptionT[List],
-    FunctionK.lift(headOption),
+    // FunctionK.lift(headOption), // Doesn't work in Scala 3
```

Comment:
Anyone know anything about this?

Reply:
That's a macro which is not implemented for Scala 3:
https://github.com/typelevel/cats/blob/main/core/src/main/scala-2.x/src/main/scala/cats/arrow/FunctionKMacros.scala

Reply:
Is it okay to disable, or maybe we should split this test into Scala 2 / Scala 3?

Reply:
I'm not sure what this is testing TBH